.github/workflows/deploy.yml

```
@@ -27,7 +27,8 @@ jobs:
     - name: Post to Splunk SOAR API
       run: |
         python -m pip install requests
-        make deploy
+        # Waiting on deployment fix
+        # make deploy
```

The deployment fix is merged, so we can un-comment this now.
```suggestion
        make deploy
```